source/common/config/grpc_mux_impl.cc

```diff
@@ -192,6 +150,54 @@ void GrpcMuxImpl::handleResponse(std::unique_ptr<envoy::api::v2::DiscoveryRespon
   queueDiscoveryRequest(type_url);
 }
 
+GrpcMuxWatchPtr GrpcMuxImpl::subscribe(const std::string& type_url,
```

Comment:
How come this code moved around? Would be ideal to keep it stable unless it needs to move for review purposes.

Reply:
Done.